db/system_keyspace.cc

```diff
@@ -3421,6 +3444,54 @@ future<> system_keyspace::save_group0_upgrade_state(service::group0_upgrade_stat
     return set_scylla_local_param(GROUP0_UPGRADE_STATE_KEY, value);
 }
 
+future<> system_keyspace::sstables_reg_create_entry(sstring location, utils::UUID uuid, sstring status, sstables::entry_descriptor desc) {
+    static const auto req = format("INSERT INTO system.{} (location, generation, uuid, status, version, format) VALUES (?, ?, ?, ?, ?, ?)", SSTABLES_REGISTRY);
```

Comment:
it would be nice to expand reg into registry. I had to jump here in order to figure out what reg stands for. but I understand it can become too long. Maybe we can remove the _entry suffix from function names.